packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountListItem.tsx

```diff
@@ -170,7 +175,7 @@ export const AccountListItem = ({
       aria-label={messages.moreOptions}
       icon={IconKebabHorizontal}
       color='default'
-      onClick={triggerPopup}
+      onClick={() => triggerPopup()}
```

Comment:
why the change here?

Reply:
I noticed that if you click the dots while the menu is open, nothing happens. So I looked at how this was being used elsewhere and updated this to match. I think ultimately the problem is a bug in the PopupMenu component, but I didn't want to dig into fixing that with this PR.